juju/docs
# Add legacy versions of the docs

## Done
Added the legacy versions of the docs to the build folder and stopped it from being removed when running `./run clean`.

## QA
- Pull this branch run
- Run `./run`
- Open http://0.0.0.0:8205/2.3/en/about-juju
- Check it looks good
- Check the versions in the menu work
- Change the URL to http://0.0.0.0:8205/1.18/en/about-juju
- See that it works and looks like the existing docs
- Run `./run clean`
- Run `./run` and check it all works still
